edit_expense/src/main/res/layout/layout_edit_expense.xml

```diff
@@ -0,0 +1,155 @@
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"/>
+
+    <EditText
+        android:id="@+id/edit_date"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:textAppearance="?textAppearanceBody1"
+        android:textColor="@color/primary_text"
+        android:layout_marginTop="@dimen/margin_small"
+        android:background="@drawable/enter_field_bg"
+        android:padding="@dimen/enter_field_padding"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@+id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@id/text_date" />
+
+    <TextView
+        android:id="@+id/text_additional_info"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/additional_info"
+        android:textColor="@color/primary_text"
+        android:layout_marginTop="@dimen/margin_large"
+        android:textAppearance="@style/TextAppearance.AppCompat.Medium"
+        app:layout_constraintTop_toBottomOf="@id/edit_date"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"/>
+
+    <EditText
+        android:id="@+id/edit_additional_info"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:textAppearance="?textAppearanceBody1"
+        android:textColor="@color/primary_text"
+        android:layout_marginTop="@dimen/margin_small"
+        android:background="@drawable/enter_field_bg"
+        android:padding="@dimen/enter_field_padding"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@+id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@id/text_additional_info" />
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/button_save"
+        style="@style/Widget.AppCompat.Button.Colored"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/save"
+        android:textSize="15sp"
+        app:strokeColor="@color/red_D7304D"
+        android:background="@color/red_D7304D"
+        app:strokeWidth="2dp"
+        app:layout_constraintTop_toTopOf="@id/button_discard"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintEnd_toStartOf="@id/button_discard"/>
+
+    <com.google.android.material.button.MaterialButton
+        android:id="@+id/button_discard"
+        style="@style/Widget.MaterialComponents.Button.OutlinedButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/discard"
+        android:textColor="@color/primary_text"
+        android:textSize="15sp"
+        app:backgroundTint="@android:color/transparent"
+        app:cornerRadius="8dp"
+        app:rippleColor="#33AAAAAA"
```

Comment:
You also have a hard coded color here. Try to maintain consistency in how you use xml resources.

Reply:
Okay, thanks alot. I'll edit it and check through other xml files

Reply:
Also you don't need to thank me after each PR comment 🙂